README.md

```diff
@@ -1 +1,103 @@
 # awesome-selfhosted-data
+
+**Status: Draft**
+
+This repository holds data used to generate https://github.com/awesome-selfhosted/awesome-selfhosted
+
+## Contributing
+
+Don't know where to start? Check issues labeled [`help wanted`](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22), [`fix`](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues?q=is%3Aissue+is%3Aopen+label%3Afix) and [`curation`](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues?q=is%3Aissue+is%3Aopen+label%3Acuration).
+
+
+### Maintenance
+
+- Software with no development activity for 6-12 months may be removed from the list
+- Unmaintained software without an active community and/or persistent security issues may be removed from the list
+- Problems should be reported automatically: [![](https://img.shields.io/travis/awesome-selfhosted/awesome-selfhosted/master?label=link%20checks)](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues/1)
+
+
+### Add new software to the list
+
+- [ ] Please submit one item per issue/pull request. This eases reviewing and speeds up the addition process.
+- [ ] Please check that your addition is not already listed at any of these lists (If it fits in one of those lists, please try to get it added there instead):
+  - [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin)
+  - [awesome-analytics](https://github.com/onurakpolat/awesome-analytics)
+  - [staticgen.com](https://www.staticgen.com/)
+  - [staticsitegenerators.net](https://staticsitegenerators.net/)
+- [ ] Please search for relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [pull requests](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
+- [ ] [Create a new `.yml` file](https://github.com/nodiscc/awesome-selfhosted-data/new/master/software) based on the template below in the `software/` directory:
+
+```yaml
+# software/my-awesome-software.yml
+name: "My awesome software" # required
+source_code_url: "https://gitlab.com/awesome/software" # required
+website_url: "https://my.awesome.softwar.e" # if different from website URL
+description: "A description of my awesome software, shorter than 250 characters." # required
+licenses: # required
+  - Apache-2.0
+  - AGPL-3.0
+platforms: # required
+  - Java
+  - Python
+  - PHP
+  - Nodejs
+  - Deb
+  - Docker
+tags: # required
+  - Automation
+  - Calendar
+  - File synchronization
+depends_3rdparty: yes # required if the software depends on a non-Free third-party service
```

Comment:
replace with a list of

```yaml
anti_features: # list of anti-features (see anti_features.yml), if applicable. Else remove this.
  - Non-Free network services
```

Possibly extend the list with more anti-features, inspired by https://f-droid.org/en/docs/Anti-Features/